src/coloring/coloring_driver.cpp

```
@@ -119,6 +121,14 @@ void do_coloring(
             undigraph.insert_edges(edges);
 
             switch (which) {
+               case MAXIMALPLANAR:
```

Cppcheck (2.21.0)

[style] The function 'visit_vertex_pair' is never used.

(unusedFunction)